### PR TITLE
Create al517-lu.md

### PR DESCRIPTION
**Pull request by al517**
### Description of Change

- Creating a file

### Test results

- No

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

File: al517-lu.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# steep garden ideas
+## growing tomatoes
```
Comment (al517) [inline al517-lu.md:2]: no clue how to grow tomatoes, sorry

File: al517-lu.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# steep garden ideas
```
Comment (al517) [inline al517-lu.md:1]: I have a few ideas

**Review by al517 — COMMENTED** [review]
review for al517 branch

**Review by githubteacher — APPROVED** [review]
👍 Looks good to me!

**Pull request merged**